app/views/layouts/application.html.slim

```diff
@@ -1,67 +1,81 @@
 doctype html
-html
+html.h-100
```

Comment:
Is this change valid and actually causing a difference? I am just surprised that a Bootstrap helper class is added to the outermost tag; I haven't seen that before.

Reply:
You are correct. This is a remaining of my previous solution to keep the header and footer intact when upgrading to bootstrap v5, but it can be implemented differently (in the layout body)